keyboards/split75/split75.c

```diff
@@ -0,0 +1,63 @@
+#include "split75.h"
+#include "rgblight.h"
+
+#include "progmem.h"
+#include "i2c_master.h"
+
+#include "action_layer.h"
+#define LED_INDICATORS
+
+extern inline void led_1_on(void);
+extern inline void led_2_on(void);
+extern inline void led_3_on(void);
+
+extern inline void led_1_off(void);
+extern inline void led_2_off(void);
+extern inline void led_3_off(void);
+
+void blink_all_leds(void);
+
+uint8_t led0 = 0, led1 = 0, led2 = 0;
+
+void indicator_init(void) {
+    // D0, D1, and D6 are the indicators and D4 is the backlight
+    DDRD |= 0x53;
+    PORTD |= 0x53;
```

Comment:
```suggestion
    setPinOutput(D0);
    setPinOutput(D1);
    setPinOutput(D6);
```
This should fix the always-on backlight, then. Bit 4 of `PORTD` was being set here.

Reply:
Hmm I've tried this but it's still on. I've set `BACKLIGHT_ENABLE = yes` in my rules.mk and I've created mappings for BL_TOGG,  BL_STEP and BL_BRTG and in my keymap. But pressing them still does nothing

Just to be clear, when I say always on - I'm talking about the underglow for the whole keyboard, not the lights on the individual keys (I don't have leds on them actually so wouldn't be able to test it)

Reply:
Yes, backlight is for the in-switch LEDs. To change the RGB underglow you would use the `RGB_` keycodes. I think this will only affect one of the halves, though.

https://github.com/showjean/ps2avrU/blob/5afaaa3035e708d7b4a3b2f88eae7f85f2895820/firmware/ps2avrGB/ledrender.c#L127-L130

Reply:
Oh I've tried `RGB_TOG` and it actually works on both halves!

Reply:
Since backlight should be working, is it worthwhile enabling it by default in rules.mk?

Reply:
Yeah, as long as it doesn't build oversize.